packages/web-components/src/components/va-combo-box/va-combo-box.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  forceUpdate,
+  Host,
+  Prop,
+  h,
+  State,
+  Fragment,
+} from '@stencil/core';
+import classnames from 'classnames';
+import { i18next } from '../..';
+import { comboBox } from './va-combo-box-library.js';
+
+/**
+ * @componentName Combo Box
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ * @guidanceHref form/combo-box
+ * @translations English
+ * @translations Spanish
+ */
+
+@Component({
+  tag: 'va-combo-box',
+  styleUrl: 'va-combo-box.scss',
+  shadow: true,
+})
+export class VaComboBox {
+  @Element() el: any;
+
+  @State() options: Array<Node>;
+
+  @State() labelNode: HTMLLabelElement;
+
+  /**
+   * Whether or not this is a required field.
+   */
+  @Prop() required?: boolean = false;
+
+  /**
+   * The combo box component will be disabled / read-only.
+   */
+  @Prop() disabled?: boolean = false;
+
+  /**
+   * Text label for the field.
+   */
+  @Prop() label!: string;
+
+  /**
+   * The placeholder string.
+   */
+  @Prop() placeholder?: string;
+
+  /**
+   * Name attribute for the select field.
+   */
+  @Prop() name!: string;
+
+  /**
+   * Selected value (will get updated on select).
+   */
+  @Prop({ reflect: true, mutable: true }) value?: string;
+
+  /**
+   * Error message to display. When defined, this indicates an error.
+   */
+  @Prop() error?: string;
+
+  /**
+   * Whether or not `aria-invalid` will be set on the inner select. Useful when
+   * composing the component into something larger, like a date component.
+   */
+  @Prop() invalid?: boolean = false;
+
+  /**
+   * Whether or not to fire the analytics events
+   */
+  @Prop() enableAnalytics?: boolean = false;
+
+  /**
+   * Optional hint text.
+   */
+  @Prop() hint?: string;
+
+  /**
+   * An optional message that will be read by screen readers when the select is focused.
+   */
+  @Prop() messageAriaDescribedby?: string;
+
+  /**
+   * The event emitted when the selected value changes
+   */
+  @Event() vaSelect: EventEmitter;
+
+  /**
+   * Whether an error message should be shown - set to false when this component is used inside va-date or va-memorable-date
+   * in which the error for the va-select will be rendered outside of va-select
```

Comment:
The documentation comments for this prop is for `va-select` and I don't think this prop is even needed for va-combo-box but just when `va-select` is being used as a child component in those other components.

For `va-combo-box`, we can probably just use the `error` prop as a falsy/truthy value to determine if the error is displayed.

Reply:
yes, this makes sense. I did indeed use `va-select` as a starting for this, and may have left this behind unintentionally. I will clean this 👍🏻